optuna/visualization/_utils.py

```diff
@@ -142,7 +142,6 @@ def _target(t: FrozenTrial) -> float:
     return filtered_trials
 
 
-def _is_reverse_scale(
-    target: Optional[Callable[[FrozenTrial], float]], direction: StudyDirection
-) -> bool:
-    return target is not None or direction == StudyDirection.MINIMIZE
+def _is_reverse_scale(study: Study, target: Optional[Callable[[FrozenTrial], float]]) -> bool:
+
+    return target is not None or study.direction == StudyDirection.MINIMIZE
```

Comment:
Yes, I think you are right. In addition, if users are unhappy with the hard-coded colourmap, they can change because retuned plots are editable. I suppose we can mention this behavior in `target` arguments of the plot functions in the documentation.

Reply:
:+1: